code/gam-national.R

```diff
@@ -35,12 +35,17 @@ error2 = error2 %>%
   mutate(northing = 221460)
 
 
-
 traffic_bam = traffic_bam %>%
   filter(count_point_id != 952939,
          count_point_id != 946853)
 traffic_bam = rbind(traffic_bam, error1, error2)
 
+# Remove counts with hour errors
+# traffic_bam %>%
+#   filter(hour == 0 | hour == 1 | hour == 3 | hour == 4 | hour == 5)
+traffic_bam = traffic_bam %>%
```

Comment:
I think this kind of data cleaning could usefully go in the `dftTrafficCounts` package at some point...